.github/sync_labels.py

```diff
@@ -408,7 +408,17 @@ def get_commits(self):
             return self._commits
 
         self._commits = self.view('commits')
-        self._commit_date = max( com['committedDate'] for com in self._commits )
+
+        # ignore merge commits with the develop branch for commit_date if needs_work is set
+        date_commits = list(self._commits)
+        if Status.needs_work.value in self.get_labels():
+            for com in date_commits:
+                message =  com['messageHeadline']
+                if message.startswith('Merge') and message.find('develop') > 0:
```

Comment:
```suggestion
                if message.startswith('Merge') and 'develop' in message:
```
I think this is more natural (and faster according to `%timeit`).